pkg/platform/runtime/target/target_test.go

```diff
@@ -15,7 +15,7 @@ func TestTrigger_IndicatesUsage(t *testing.T) {
 		},
 		{
 			"Reset exec does not count as usage",
-			TriggerResetExec,
+			TriggerRefresh,
```

Comment:
I think we do want to count this as runtime use. Regardless of what this ends up being, please update the documentation string in the line above.